core/src/main/java/com/linecorp/armeria/client/UserClient.java

```diff
@@ -126,26 +125,24 @@ protected final Endpoint endpoint() {
      */
     protected final O execute(HttpMethod method, String path, @Nullable String query, @Nullable String fragment,
                               I req, BiFunction<ClientRequestContext, Throwable, O> fallback) {
-        return execute(null, endpoint, method, path, query, fragment, req, fallback);
+        return execute(endpoint, method, path, query, fragment, req, fallback);
     }
 
     /**
      * Executes the specified {@link Request} via {@link #delegate()}.
-     *
-     * @param eventLoop the {@link EventLoop} to execute the {@link Request}
-     * @param endpoint the {@link Endpoint} of the {@link Request}
+     *  @param endpoint the {@link Endpoint} of the {@link Request}
```

Comment:
nit: double space here. 
```suggestion
     * @param endpoint the {@link Endpoint} of the {@link Request}
```